homeworks/03-data-structure/src/binary-tree/binaryTree.ts

```diff
@@ -0,0 +1,67 @@
+import { assertNever } from '../utils/commonUtils';
+
+export type TreeNode<T> = {
+  value: T;
+  left: TreeNode<T> | null;
+  right: TreeNode<T> | null;
+};
+
+export enum TraverseType {
+  depthFSInOrder = 'DFS_INORDER',
+  depthFSPreOrder = 'DFS_PREORDER',
+  depthFSPostOrder = 'DFS_POSTORDER',
+  breadthFS = 'BFS',
```

Comment:
Still doesn't correspond to the Enum naming convention 😅 (should be PascalCase)

I would name it somehow like: `DFSInOrder`, `BFS`, etc.

Reply:
Uh oh - https://github.com/vbarbasins/evo-ts-bootcamp/pull/3/commits/bc7b12531c1698172ad915b41e66b484ce5bb1b7